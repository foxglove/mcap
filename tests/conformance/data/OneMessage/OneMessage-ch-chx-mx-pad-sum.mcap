version https://git-lfs.github.com/spec/v1
oid sha256:f544dbe302d319e3192caf66329825a2d4da90f2e372b37054bdca37a340f793
size 394
