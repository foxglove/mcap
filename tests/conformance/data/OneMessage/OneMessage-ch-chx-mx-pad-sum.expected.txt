Header profile="" library=""
ChannelInfo id=1 topic=example message_encoding=a schema_encoding=c schema=b schema_name=Example metadata={foo=bar}
Message channel_id=1 sequence=10 publish_time=1 log_time=2 message_data=<010203>
MessageIndex channel_id=1 records={2=66}
ChunkIndex start_time=2 end_time=2 chunk_start_offset=28 chunk_length=139 message_index_offsets={1=169} message_index_length=34 compression="" compressed_size=100 uncompressed_size=100
SummaryOffset group_opcode=12 group_start=219 group_length=0
SummaryOffset group_opcode=9 group_start=219 group_length=0
SummaryOffset group_opcode=7 group_start=219 group_length=90
Footer summary_start=219 summary_offset_start=309
